### PR TITLE
Fixed #7 Inconsistancy

### LINKED ISSUE #7: Spec Correction: Fix Place Character
Place does not correlate with whats  in the code

### PR DESCRIPTION
**Pull request by darkliahos**

### CODE REVIEW

File: LanguageSpec.md [hunk 1]
```diff
@@ -116,13 +116,17 @@ So it would be:
 
 ### Place Character
 
-If we wanted to move the character sprite anywhere on the screen, we could just type
+If we wanted to move the character sprite anywhere on the screen with scaling, we could just type
 
-```PLACE [Character] ([Up, Down, Left, Right])```
+```PLACE [Character] ([x, y, Scale-Height, Scale-Width])```
```
Comment (tsurinssword) [inline LanguageSpec.md]: is the scaling optional or will the user have to supply scale every time? 
If it's every time, that's increases the amount of typing the user has to do for every PLACE command.

Comment (tsurinssword) [reply LanguageSpec.md]: Never mind, should've read on first.

File: LanguageSpec.md [hunk 2]
```diff
@@ -116,13 +116,17 @@ So it would be:
...
 
-So it would be:
+So if we were to place Alanis in the top left corner of the screen at 50px height and 20px width, we would write
+
+```PLACE [Alanis] (0,0,50,20)```
+
+If we did not want to worry about scaling we could just write:
```
Comment (tsurinssword) [inline LanguageSpec.md]: I agree with this as I'm planning to make the underlying draw method work with both supplied scale and default scale values when no scale has been supplied.

Comment (darkliahos) [reply LanguageSpec.md]: Awesome, glad we are thinking on the same page.

**Pull request merged**